src/tests/integration-test

```diff
@@ -405,13 +405,11 @@ class UDisksTestCase(unittest.TestCase):
                 else:
                     break
 
-        # check localstatedir
+        # create localstatedirs
         for d in (os.path.join(make_vars['localstatedir'], 'run', 'udisks2'),
                   os.path.join(make_vars['localstatedir'], 'lib', 'udisks2')):
             if not os.path.exists(d):
-                sys.stderr.write('The directory %s does not exist; please '
-                                 'create it before running these tests.\n' % d)
-                sys.exit(1)
+                os.mkdir(d)
```

Comment:
Maybe we could do this only when `JENKINS_HOME` variable is set? But creating some directories on a machine where you run UDisks test suite (so it probably is either in CI or on a development machine) should be ok anyway.

Reply:
I'm not even sure whether these directories are ever used for anything nowadays. The state file storage has changed a little bit since this test was written.